src/sensor.cpp

```diff
@@ -420,7 +420,7 @@ namespace librealsense
                             _source.invoke_callback(std::move(pref));
                     }
                 },
-                static_cast<int>(_source.get_published_size_option()->query()));
+                DEFAULT_FRAME_BUFFERS);
```

Comment:
If the backend requires 4 buffers to avoid frame drops then what published_queue_size could be updated to {4,32(16),4,1}